src/types/qwseat.cpp

```diff
@@ -279,14 +288,22 @@ bool QWSeat::pointerHasGrab() const
     return wlr_seat_pointer_has_grab(handle());
 }
 
-void QWSeat::pointerNotifyAxis(uint32_t time_msec, wlr_axis_orientation_t orientation, double value, int32_t value_discrete, wlr_axis_source_t source)
+void QWSeat::pointerNotifyAxis(uint32_t time_msec, wlr_axis_orientation_t orientation, double value, int32_t value_discrete, wlr_axis_source_t source, wl_pointer_axis_relative_direction relative_direction)
```

Comment:
don't need wl_pointer_axis_relative_direction in wlroots 0.17, you
 need add a new function